include/deal.II/base/time_stepping.templates.h

```diff
@@ -393,6 +393,29 @@ namespace TimeStepping
 
     switch (method)
       {
+        case (FORWARD_EULER):
+          {
+            this->n_stages = 1;
+            this->b.reserve(this->n_stages);
+            this->b.push_back(1.0);
+
+            std::vector<double> tmp;
+            tmp = {{0.0}};
```

Comment:
Why not initialize right at the point of declaring the variable? Then you can even make the variable `const`. Same below of course.